modules/firelens/main.tf

```diff
@@ -0,0 +1,35 @@
+module "log_router_container" {
+  source = "../../modules/container_definition"
+  image  = "wellcome/fluentbit:edge"
```

Comment:
Must update this with fixed tag of image when https://github.com/wellcomecollection/dockerfiles/pull/90 builds

Reply:
tag is 134